test/js/samples/capture-inject-dev-only/expected.js

```diff
@@ -25,7 +25,7 @@ function create_fragment(ctx) {
 	return {
 		c() {
 			p = element("p");
-			t0 = text(ctx.foo);
+			t0 = text(ctx.foo + "");
```

Comment:
it'd be good if we didn't have the `+ ""` here, since it's coerced to a string anyway